[CDS-100013] CD Release notes 1.52.4 (#7705)
harness/developer-hub: release-notes/continuous-delivery.md

## August 2024

### Version 1.52.4

#### Fixed issues

- The error title **Intervention** was displayed when the verification step failed. The title is now updated to **Verification Failure**. (CDS-99671,ZD-65113)

> - The Queries in `verificationJobInstances` scanned too many documents and raised the CPU utilization. This issue has been resolved now by adding more `verificationJobInstances` indexes. (CDS-99540)

Reviewer: ```suggestion
- The queries in `verificationJobInstances` scanned too many documents and raised CPU utilization. This issue has been resolved now by adding more `verificationJobInstances` indexes. (CDS-99540)
```

> - The runtime input was displayed even when a **fixed values** was selected from the templates in the monitored services. This issue has been resolved since `defaultValueToReset` field is passed to the MultiTypeService and MultiTypeEnvironment fields. (CDS-99518) 

Reviewer: ```suggestion
- The runtime input was displayed even when **fixed values** was selected from the templates in the monitored services. This issue has been resolved since `defaultValueToReset` field is passed to the MultiTypeService and MultiTypeEnvironment fields. (CDS-99518) 
```

- On the services page, the deployment type icon did not consistently appear. This issue has been resolved. (CDS-99331, ZD-66892)

> - The GitOps deployment were not tracked for multiple projects as the Harness Gitops instances service was out of sync with the applications without the project mappings. This issue has been resolved. (CDS-98989, ZD-63203)

Reviewer: ```suggestion
- GitOps deployments were not tracked for multiple projects because the Harness Gitops instances service was out of sync with applications without project mappings. This issue has been resolved. (CDS-98989, ZD-63203)
```

> - The error message associated with the GitHub Connector parameters was ambiguous. This issue was resolved by improving the response message for missing parameters in GitHub Connector. (CDS-97760)

Reviewer: ```suggestion
- An error message associated with missing parameters in the GitHub Connector was ambiguous. This issue was resolved. (CDS-97760)
```

## August 2024

### Version 1.52.4

#### Fixed issues

- The error title **Intervention** was displayed when the verification step failed. The title is now updated to **Verification Failure**. (CDS-99671,ZD-65113)

> - The queries in `verificationJobInstances` scanned too many documents and raised CPU utilization. This issue has been resolved now by adding more `verificationJobInstances` indexes. (CDS-99540)

Reviewer 2: This is an internal error, we can remove this from release notes.

> - The runtime input was displayed even when **fixed values** was selected from the templates in the monitored services. This issue has been resolved since `defaultValueToReset` field is passed to the MultiTypeService and MultiTypeEnvironment fields. (CDS-99518) 

Reviewer 2: ```suggestion
- The runtime input symbol was getting displayed even when **fixed values** was selected from the templates in the monitored services. This issue has been resolved. (CDS-99518) 
```
